### DISCUSSION TITLE
HTLM Export not formatting well wiki links, numbered lists and tags

### DISCUSSION THREAD

**Discussion by FamedBear16** [mokeyish/obsidian-enhancing-export#185] [Q&A]
HI,
When I convert to HTML, words that are supposed to be on a different line, they appear in the same.
This happens with wiki links [[ ]] and numbered lists, #tags

For example:
[[file 1]]
[[file 2]]
1. mario
2. gino
#tests 
mario

is render in HTML as
file 1 file 2
1. mario 2. gino
#test mario

**Comment by bwat47** [comment]
Try adding this to your pandoc arguments:

`-f markdown+hard_line_breaks`
